decimal_test.py

```diff
@@ -0,0 +1,23 @@
+import crdch_model as cm
+from linkml_runtime.dumpers import yaml_dumper
+
+
+def test_dumps_vcc():
+    vcc_obs = cm.Observation(observation_type=cm.CodeableConcept(), value_codeable_concept=cm.CodeableConcept())
+    vcc_dump = yaml_dumper.dumps(vcc_obs)
+    # print statements ignored by default in passing pytests?
+    # https://stackoverflow.com/questions/24617397/how-to-print-to-console-in-pytest
+    # try -s argument or PyCharm specific configuration?
+    print(vcc_dump)
+    assert len(vcc_dump) > 0
+
+
+def test_dumps_decimal():
+    decimal_obs = cm.Observation(observation_type=cm.CodeableConcept(), value_decimal="1.23")
+    vd_dump = ""
```

Comment:
I don't think you need this line -- AFAIK, `try` and `with` [doesn't create a scope in Python](https://stackoverflow.com/a/6167952/27310), so any variables declared inside those sections are available outside of it.

Reply:
OK, I was exploring ways to show that the test failed because an exception was raised, not because the code was syntactically OK but just did the wrong thing/returned the wrong value. I don't think I was trying to create a scope.

Reply:
Ah, no worries, I misunderstood what you were doing here! I'll resolve this conversation.